services/authenticationService.js

```diff
@@ -2,7 +2,7 @@ class userService {
     constructor() {
```

Comment:
classes prefer to be named with the uppercase letter `UserService`

Reply:
Yes, I forgot about it when created the file. I've changed it.